docs/developer-guide/interactivity.md

```diff
@@ -1,8 +1,72 @@
 # Adding Interactivity
 
-> This article discusses interacting with data (i.e. selecting, or picking objects). Viewport controls (panning, zooming etc) are discussed in [Controllers](/docs/api-reference/map-controller.md).
+## Controlling the camera
```

Comment:
Should we capitalize Camera for consistency? I get that there are strong opinions on whether or not to do so, but at least each page should consistent.